bin/grouper-fe

```diff
@@ -66,6 +66,8 @@ def main(argv):
     parser.add_argument("-V", "--version", action="version",
                         version="%%(prog)s %s" % grouper.__version__,
                         help="Display version information.")
+    parser.add_argument("-a", "--address", type=str, default="",
```

Comment:
i believe it does support `null` but yeah, the convention seems to be leaving it blank. and i'm the one who's nit picking so if anything i'm making this all too complicated. :)